docs/manual/annotating-libraries.tex

```diff
@@ -536,8 +536,11 @@
 a warning whenever a stub file specification is redundant with
 bytecode annotations.
 
-A file being compiled takes precedence over a stub file.  If file \<A.java>
-is being compiled, then any stub for class \<A> is ignored.
+By default, a file being compiled takes precedence over a stub file:  if
+file \<A.java> is being compiled, then any stub for class \<A> is ignored.
+The command-line option \<-AmergeStubsWithSource> tells the checker to
+choose the stronger type from stub files and source files when both are
```

Comment:
What does "stronger" mean? If this means the subtype, this doesn't seem usable for e.g. the Nullness Checker: the default is the stronger annotation (`@NonNull`) and therefore a stubfile couldn't provide the weaker (`@Nullable`).
If this only meant for WPI, this description should be moved there (and the flag guarded by checking for WPI.)

Reply:
Stronger does mean subtype. The only actual use case that I know of for this flag is WPI, but I imagine in theory it could be useful for someone who wanted to store annotations in stub files instead of in the source directly. I don't think it's a good idea to guard it behind a WPI flag for that reason.

You're right that this won't work for the Nullness Checker or any other type system where the default type isn't top. I will clarify the documentation.